docs/fargate.md

```diff
@@ -115,6 +120,143 @@ fi
 
 `EFS_ID` is going to be used in setting up Volumes for metric pods.
 
+## Logs
+
+The following are some of the steps needed to setup and enable logs collection on Fargate
+
+### Fluent-bit log router configuration
+
+Amazon EKS on Fargate offers a built-in log router based on Fluent Bit. This means that you don't explicitly run a Fluent Bit container as a
+sidecar, but Amazon runs it for you. All that you have to do is configure the log router.
+
+Below are the steps to configure the log router on fargate
+
+- Create a dedicated Kubernetes namespace named `aws-observability`
+
+```yaml
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+```
+
+The fargate log router manages the `Service` and `Input` sections. One needs to create a `ConfigMap` to enable log routing to cloudwatch, an
+example is shown below:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  flb_log_cw: "true" # Set to true to ship Fluent Bit process logs to CloudWatch.
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch_logs
+        Match   kube.*
+        region region-code
+        log_group_name my-logs
+        log_stream_prefix from-fluent-bit-
+        log_retention_days 60
+        auto_create_group true
+```
+
+You can stream logs from Fargate directly to Amazon CloudWatch, using the output plugin shown above. Cloudwatch is currently the only
+supported output plugin in fluent-bit running on EKS fargate.
+
+For more information on this refer to [fargate-logging](https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html)
```

Comment:
We should document or note that the above steps are not enough due to `Prerequisites`

![image](https://user-images.githubusercontent.com/58699848/232710418-5a97a318-e7c6-4425-a638-0a9d89f1f198.png)

We should add some troubleshooting for customers which do not followed the `prerequisites`. I would be also nice to have some verification steps

Reply:
> We should document or note that the above steps are not enough due to Prerequisites

I see we do it further in this doucument.

Reply:
I'll add a prerequisites section in the log router configuration.

Reply:
Added additional troubleshooting steps in the last commit